client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -40,6 +70,20 @@ class RelatedPostsEdit extends Component {
 		const displayPosts = postsToDisplay.slice( 0, postsToShow );
 		const previewClassName = 'related-posts__preview';
 
+		const displayMockupPosts = posts.length ? false : true;
+
+		const inlineMockupPosts = [];
+		for ( let i = 0; i < postsToShow; i++ ) {
+			inlineMockupPosts.push(
+				<MockupPostEdit
+					key={ i.toString() }
+					displayThumbnails={ displayThumbnails }
+					displayDate={ displayDate }
+					displayContext={ displayContext }
+				/>
+			);
+		}
```

Comment:
fun fact! we don't need the `for` loop or the mutation if we don't want them…

```js
const inlineMockupPosts = postsToShow.map( ( post, key ) => (
	<MockupPostEdit { ...{ key, displayThumbnails, displayDate, displayContext } } />
) );
```

Reply:
Good call! Will have a look at adapting to this way.

Reply:
Just to reflect on the comment below, `postsToShow` is an integer and a `for` loop is probably the appropriate solution here.